src/test/java/uk/gov/hmcts/reform/sscs/service/AirLookRcSpreadSheetTest.java

```diff
@@ -93,12 +93,10 @@ public void testAllPostcodesHavePip() {
         List<String> missingPipVenuePostcodes = new ArrayList<>();
 
         Set<String> rcKeys = lookupData.keySet();
-        Iterator<String> iterator = rcKeys.iterator();
-        while (iterator.hasNext()) {
-            String postcode = iterator.next();
-            if (!venueData.keySet().contains(postcode) && !lookupData.get(postcode).equals("Glasgow")) {
-                if (!realPostcodesWithNoVenue.contains(postcode)  && !notRealPostcodes.contains(postcode)) {
-                    missingPipVenuePostcodes.add(postcode);
+        for (String rcKey: rcKeys) {
+            if (!venueData.containsKey(rcKey) && !lookupData.get(rcKey).equals("Glasgow")) {
+                if (!realPostcodesWithNoVenue.contains(rcKey)  && !notRealPostcodes.contains(rcKey)) {
+                    missingPipVenuePostcodes.add(rcKey);
```

Comment:
if you wanted to write this with  java 8 it could be like this, you can keep it this way if you like both are fine ;
        rcKeys.stream()
                .filter(rcKey -> !venueData.keySet().contains(rcKey) &&
                                 !lookupData.get(rcKey).equals("Glasgow") &&
                                 !realPostcodesWithNoVenue.contains(rcKey)  &&
                                 !notRealPostcodes.contains(rcKey))
                .forEach(postcode->missingPipVenuePostcodes.add(postcode));

Reply:
@MohammedHMCTS I also think it's better to use 'postCode' instead of 'rcKeys'. What's an rcKey?

Reply:
@MohammedHMCTS minor but could you put a space before the colon in the for () statement? same as below